app/src/test/java/io/apicurio/registry/RegistrySerdeTest.java

```diff
@@ -209,6 +208,33 @@ public void testAvro(Supplier<RegistryService> supplier) throws Exception {
         }
     }
 
+    @RegistryServiceTest
+    public void testAvroJSON(Supplier<RegistryService> supplier) throws Exception {
+        Schema schema = new Schema.Parser().parse("{\"type\":\"record\",\"name\":\"myrecord3\",\"fields\":[{\"name\":\"bar\",\"type\":\"string\"}]}");
+        try (AvroKafkaSerializer<GenericData.Record> serializer = new AvroKafkaSerializer<GenericData.Record>(supplier.get());
+             Deserializer<GenericData.Record> deserializer = new AvroKafkaDeserializer<>(supplier.get())) {
+            HashMap<String, String> config = new HashMap();
+            config.put(AvroEncoding.AVRO_ENCODING, AvroEncoding.AVRO_JSON );
+            serializer.configure(config,false);
+            deserializer.configure(config, false);
+            serializer.setGlobalIdStrategy(new AutoRegisterIdStrategy<>());
+
+            GenericData.Record record = new GenericData.Record(schema);
+            record.put("bar", "somebar");
+
+            String subject = generateArtifactId();
+
+            byte[] bytes = serializer.serialize(subject, record);
```

Comment:
I think it would be good to check these bytes here to ensure the data has been encoded as json. I think this test would currently still pass if the encoding was avro-binary.

Reply:
Agreed.